### ISSUE TITLE
Neptune audit logs to CloudWatch

### ISSUE THREAD

**Issue by ewbankkit** [terraform-providers/terraform-provider-aws#9094]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

You can now configure Amazon Neptune to publish audit logs to Amazon CloudWatch Logs.

### New or Affected Resource(s)

* [`aws_neptune_cluster`](https://www.terraform.io/docs/providers/aws/r/neptune_cluster.html)

### Potential Terraform Configuration

```hcl
resource "aws_neptune_cluster" "example" {
  enabled_cloudwatch_logs_exports = [
    "audit",
  ]
}
```

### References

[Announcement](https://aws.amazon.com/about-aws/whats-new/2019/06/you-can-now-publish-amazon-neptune-audit-logs-to-cloudwatch/).

**Comment by ericksoen** [comment]
It looks like this is now supported in the [aws-sdk-go Neptune API](https://github.com/aws/aws-sdk-go/blob/master/service/neptune/api.go#L6705). I'd be interested in taking a look at this, although it's not clear anywhere what the valid values other than audit are.

**Comment by bflad** [comment]
Support for the `enable_cloudwatch_logs_exports` argument in the `aws_neptune_cluster` resource has been merged and will release with version 2.49.0 of the Terraform AWS Provider, Thursday this week. Thanks to @ericksoen for the implementation. 👍
